api.py

```diff
@@ -0,0 +1,66 @@
+import json
+import six
+from pywinauto.backend import Singleton
+from pywinauto.actionlogger import ActionLogger
+
+from .injector import Injector
+from .channel import Pipe
+
+# backend exit codes enum
+OK=0
+PARSE_ERROR=1
+UNSUPPORTED_ACTION=2
+MISSING_PARAM=3
+RUNTIME_ERROR=4
+NOT_FOUND=5
+UNSUPPORTED_TYPE=6
+
+class InjectedBackendError(Exception):
+    """Base class for exceptions on the injected backend side"""
+    pass
+
+class UnsupportedActionError(InjectedBackendError):
+    pass
+
+class BackendRuntimeError(InjectedBackendError):
```

Comment:
I'd suggest replace `Backend` to `Target` in class names to avoid confusion with pywinauto's backends.

Reply:
Done